test_communication/package.xml

```diff
@@ -11,21 +11,23 @@
   <maintainer email="dthomas@osrfoundation.org">Dirk Thomas</maintainer>
   <license>Apache License 2.0</license>
 
+  <build_depend>ament_cmake_auto</build_depend>
+  <build_depend>builtin_interfaces</build_depend>
+  <build_depend>rclcpp</build_depend>
+  <build_depend>rclpy</build_depend>
+  <build_depend>rcl</build_depend>
```

Comment:
The generated type support triggered by `rosidl_generate_interfaces` doesn't depend on the client libraries `rcl*`. The only build dependency necessary for that is `rosidl_default_generators`.

Also `rosidl_default_runtime` should be a run dependency.